app/forms/housing_costs_form.rb

```diff
@@ -0,0 +1,38 @@
+class HousingCostsForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include SessionPersistable
+  include NumberValidatable
+
+  ATTRIBUTES = %i[housing_payments housing_payments_frequency housing_benefit_value housing_benefit_frequency].freeze
+  HOUSING_PAYMENT_FREQUENCIES = %w[every_week every_two_weeks every_four_weeks monthly total].freeze
```

Comment:
Rather than repeat this constant here, it might be better to delete this and instead reference OutgoingsForm::VALID_FREQUENCIES. Likewise with MortgageOrLoanPaymentForm::HOUSING_PAYMENT_FREQUENCIES